components/mdns/mdns_networking_lwip.c

```diff
@@ -338,11 +338,15 @@ size_t _mdns_udp_pcb_write(mdns_if_t tcpip_if, mdns_ip_protocol_t ip_protocol, c
     }
     memcpy((uint8_t *)pbt->payload, data, len);
 
+    ip_addr_t tmp;
```

Comment:
Nitpick. Let's just call it `ip_addr` rather than `temp`...
```suggestion
    ip_addr_t ip_add_copy;
```